.github/TRANSLATION.md

```
@@ -0,0 +1,70 @@
+
+## Getting Started
+
+### 1. Create a Locize Account
+
+If you don't already have an account, please register using our invite link:
+
+[Register at Locize](https://www.locize.app/register?invitation=t1VDfqoRvj8eUkd1JasxxrBCCI4SAqeeofa2YumAgmVDRxkr4vO1jKqNmpaNCv7H)
+
+This invitation will give you access to our translation project once you’ve created your account.
+
+
```

markdownlint-cli2 (0.17.2)

Multiple consecutive blank lines
Expected: 1; Actual: 2

(MD012, no-multiple-blanks)